src/Oscoin/Crypto/Hash/Mock.hs

```diff
@@ -34,9 +34,10 @@ import           Text.Read (readMaybe)
 import qualified Text.Show as Show
 import           Web.HttpApiData (FromHttpApiData(..))
 
--- For our mock implementation we use a fast, non-cryptographic hash
--- See: https://cyan4973.github.io/xxHash/
-import           Data.Digest.XXHash.FFI
+-- For the mock crypto we use the FNV-1 hashing function.
+-- Unfortunately using something faster like xxhash would generate
+-- too many collisions.
+import           Data.ByteArray.Hash
```

Comment:
Can you explicit import `FnvHash` from here? Not clear ideas where it's coming from :smile:

Reply:
Yup, by all means! I was trying as a last valiant attempt to get rid of the final space spike, but no dice so far, and will stop soon anyway as I want to move to #402 quite quickly. My intuition is that there is still _something_ being retained and not fully evaluated when we insert a block in the pure `BlockStore`, but so far deliberate use of NFData didn't yield the expected result. Oh well.

Reply:
149d81d